packages/contracts-bedrock/test/setup/CommonTest.sol

```diff
@@ -42,17 +43,18 @@ contract CommonTest is Test, Setup, Events {
     IOptimismMintableERC20Full L2Token;
     ILegacyMintableERC20Full LegacyL2Token;
     ERC20 NativeL2Token;
-    ERC20 BadL2Token;
     IOptimismMintableERC20Full RemoteL1Token;
 
     function setUp() public virtual override {
+        // Because Setup.setup() may create a forked network, all cheatcodes must be run prior to any
+        // state modifying cheat codes.
```

Comment:
nit: I think this is saying that state changes from state modifying cheats won't persist on the fork created by `Setup.setUp`? But it reads a bit confusingly, would be nice to clarify this

Reply:
Yes, that's the intent but this was kind of a nonsense sentence.

```suggestion
        // Setup.setup() may switch the tests over to a newly forked network. Therefore
        // state modifying cheatcodes must be run after Setup.setup(), otherwise the 
        // changes will not be persisted into the new network. 
```

Reply:
Making that change. 25573b8aa8e0